DEVELOPING.md

```diff
@@ -0,0 +1,53 @@
+Guidelines for Developers of OpenSim-Core
+===========================================
+OpenSim is a community resource that is housed in the OpenSim-Core repository.
```

Comment:
Most of GitHub is case sensitive. I would use the case-correct name for the repo to avoid confusion. It is `opensim-core`.